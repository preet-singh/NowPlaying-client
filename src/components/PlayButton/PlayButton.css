.FixedBar .fixed-play-button {
  margin-left: 30px;
  background: none;
  /* background: url('https://i.imgur.com/IYOSQ5U.png') no-repeat; */
}

.play {
  color: white;
  margin-left: 10px;
  cursor: pointer;
}

.play:hover {
  color: #00d838;
}