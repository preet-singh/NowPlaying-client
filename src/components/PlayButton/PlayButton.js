import React from 'react';
import './PlayButton.css'

function PlayButton(props) {
  return (
    <button type='button'>
      Play
    </button>
  )
}