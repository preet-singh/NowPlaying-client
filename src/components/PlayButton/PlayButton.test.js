import React from 'react';
import PlayButton from './PlayButton.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`Playbutton component`, () => {

  it('renders the playbutton', () => {
    const wrapper = shallow(<PlayButton />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})