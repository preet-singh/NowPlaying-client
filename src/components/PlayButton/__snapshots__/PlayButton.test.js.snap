// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playbutton component renders the playbutton 1`] = `
<button
  className="fixed-play-button"
  onClick={[Function]}
  type="button"
>
  <FontAwesomeIcon
    border={false}
    className="play"
    fixedWidth={false}
    flip={null}
    icon={
      Object {
        "icon": Array [
          448,
          512,
          Array [],
          "f04c",
          "M144 479H48c-26.5 0-48-21.5-48-48V79c0-26.5 21.5-48 48-48h96c26.5 0 48 21.5 48 48v352c0 26.5-21.5 48-48 48zm304-48V79c0-26.5-21.5-48-48-48h-96c-26.5 0-48 21.5-48 48v352c0 26.5 21.5 48 48 48h96c26.5 0 48-21.5 48-48z",
        ],
        "iconName": "pause",
        "prefix": "fas",
        "type": "icon",
      }
    }
    inverse={false}
    listItem={false}
    mask={null}
    pull={null}
    pulse={false}
    rotation={null}
    size={null}
    spin={false}
    swapOpacity={false}
    symbol={false}
    title=""
    transform={null}
  />
</button>
`;
